.github/workflows/dotnet-ci.yml

```
@@ -0,0 +1,51 @@
+            /d:sonar.host.url="https://sonarcloud.io" `
+            /d:sonar.sources="QRCodeBasedMetroTicketingSystem\" `
+            /d:sonar.exclusions="**/bin/**,**/obj/**,**/*.Designer.cs,**/Migrations/**,**/*.cshtml,**/*.scss,**/*.css,**/*.js,**/*.cshtml.cs" `
+
+      - name: Build solution
+        run: dotnet build QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.sln --no-restore --configuration Release --no-incremental
+
+      - name: End SonarCloud analysis
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: dotnet sonarscanner end /d:sonar.token="${{ secrets.SONAR_TOKEN }}"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)